index.js

```diff
@@ -1,95 +1,123 @@
 // ***** Question 1 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
-
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+console.log("%cQuestion 1", "color: red") 
+let counter = 1
+counter = 2
+console.log(counter) 
+// => 2
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+console.log("%cQuestion 2", "color: red") 
 
-// console.log(name)
-// // => "Ian"
-// name = "Not Ian" 
-// // => TypeError
-// console.log("%c----------", "color: red") 
+const name = "Warren"
+console.log(name)
+// => "Ian"
+name = "Not Ian" 
+// => TypeError
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red") 
+function drinkWater() {
+  let currentThirstLevel = 12
+  console.log("Man I sure am thirsty")
+    currentThirstLevel -= 1
+    console.log("Ahh that hits the spot")
+    return currentThirstLevel
+}
```

Comment:
For this one, the idea is to have `drinkWater` take in `currentThirstLevel` as an argument instead of declaring it inside the function:

```js
function drinkWater(currentThirstLevel) {
  console.log("Man I sure am thirsty")
    currentThirstLevel -= 1
    console.log("Ahh that hits the spot")
    return currentThirstLevel
}
```

That way, when we call it, currentThirstLevel will be assigned to whatever value we want:

```js
drinkWater(12)
```